web/ui/static/js/targets.js

```diff
@@ -21,12 +21,16 @@ function showUnhealthy(_, container) {
 }
 
 function init() {
-  if (!localStorage.selectedTab || localStorage.selectedTab == "all-targets"){
-    $("#all-targets").parent().addClass("active");
+  if (document.URL.includes('targets')) {
```

Comment:
Rather that checking the URL, it should be possible to check `$("#unhealthy-targets").length`: if > 0 then apply `localStorage.selectedTab` preferences otherwise show everything. What do you think?

Reply:
sure. that would work in a better way too. Thanks for the suggestion. Implementing..